interface/patient_file/encounter/Tests/Service/ViewHelperTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Does it really make sense to create a script that does nothing. Can't this be added when there is an actual test?